lib/kubeclient/application_default_credentials.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'googleauth'
+
+module Kubeclient
+  # Get a bearer token from the Google's application default credentials.
+  class ApplicationDefaultCredentials
```

Comment:
This name sounds pretty generic, perhaps should somehow include "Google".

Reply:
Done 24145e9. I thought that the name was already pretty long... ;) But I agree.